.github/workflows/react-native-code-push-ci.yml

```diff
@@ -27,12 +27,23 @@ jobs:
       - run: adb shell settings put global window_animation_scale 0.0
       - run: adb shell settings put global transition_animation_scale 0.0
       - run: adb shell settings put global animator_duration_scale 0.0
+      - name: Setup Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'microsoft'
+          java-version: '11'
       - name: Package Installation
-        run: npm install
+        run: npm i
+      - name: Install react-native
+        run: npm install -g react-native
+      - name: Setup Android tests
+        run: npm run build:tests && npm run test:setup:android
+      - name: Remove react-native
+        run: npm remove -g react-native
```

Comment:
Why do we need to remove the react-native?

Reply:
It is a workaround for the next error: ```ReferenceError: SHA-1 for file /metro-runtime/src/polyfills/require.js) is not computed.```

Taken from here: https://github.com/facebook/metro/issues/330#issuecomment-572917876